client/deposits/list/index.js

```diff
@@ -74,10 +74,8 @@ export const DepositsList = () => {
 		getQuery()
 	);
 
-	const columnsArgs = [
```

Comment:
I know this was a neat thing to do, but the "exhaustive deps" rule cannot determine the dependencies when used this way.